sports_association/models/sport_issue.py

```diff
@@ -27,6 +27,8 @@ class SportIssue(models.Model):
 
     tag_ids = fields.Many2many('sport.issue.tag', string='Tags')
 
+    to_do_action_ids = fields.One2many('sport.to.do.action', 'issue_id', string='To do actions')
```

Comment:
Por convención, si por ejemplo nuestro modelo es sport.issue y queremos hacer que otro modelo cuelgue de este, se suele poner de nombre de modelo la primera parte del anterior:
Por ejemplo sport.issue.action o sport.issue.to.do.action